doc/setup/quickstart_cmake/index.md

```diff
@@ -191,6 +191,33 @@ If you create a new `cpp` file and want to compile it, you need to add another `
 For example, after adding `another_program.cpp` your `CMakeLists.txt` may look like this:
 \snippet test/external_project/seqan3_setup_tutorial/CMakeLists.txt adding_files
 
+# Including SeqAn3 as external project
+
+```cmake
+cmake_minimum_required (VERSION 3.14)
+
+project (my_app LANGUAGES CXX VERSION 1.0.0)
+
+set (seqan3_git_tag "3.2.0")
```

Comment:
This will "quickly" become outdated. We should add a note in the release note that we should adapt this here after releasing.

Reply:
COuld also just be a `# Adapt as needed.`?

Reply:
That would be better.
```suggestion
set (seqan3_git_tag "#.#.#") # adapt as needed, e.g. "3.2.0"
```